### ISSUE TITLE
failed to download taxon_hierarchies from opendata.eol.org

### ISSUE THREAD

**Issue by jhpoelen** [jhpoelen/eol-globi-data#274]
To reduce load on EOL api to resolve taxon names, common names and key images, the eol team is working on providing a taxon hierarchy archive. 

Today, I unsuccessfully attempted to download EOL taxon hierarchies from http://opendata.eol.org using wget:

```
 wget http://opendata.eol.org/hierarchy_entries.tgz
--2017-01-12 15:59:55--  http://opendata.eol.org/hierarchy_entries.tgz
Resolving opendata.eol.org (opendata.eol.org)... 160.111.248.28
Connecting to opendata.eol.org (opendata.eol.org)|160.111.248.28|:80... connected.
HTTP request sent, awaiting response... 503 Service Unavailable
2017-01-12 15:59:57 ERROR 503: Service Unavailable.
```

@jhammock please let me know if there's any more information you need to troubleshoot this issue

**Comment by jhpoelen** [comment]
Today, I motivated myself to make another attempt at downloading taxon_hierarchies.tgz . This time, it left me with a **partial** download, similar to my experience using a mouse click to download the file using a browser. See logs below. Could it be that the service simply disconnects connections that last longer than 10 mins?

@jhammock did you ever consider using a service like amazon's s3 (or similar) to cough up the opendata.eol.org files. It could be as simple as creating a redirect rule in nginx.

```
curl 'http://opendata.eol.org/hierarchy_entries.tgz' -H 'Accept-Encoding: gzip, deflate, lzma, sdch' -H 'Accept-Language: en-US,en;q=0.8' -H 'Upgrade-Insecure-Requests: 1' -H 'User-Agent: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/55.0.2883.87 Safari/537.36 OPR/42.0.2393.94' -H 'SE-Proxy-Authorization: [146 bytes were stripped]' -H 'Accept: text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8' -H 'Referer: http://opendata.eol.org/' -H 'Cookie: __utma=213292672.677802574.1481607151.1484675887.1484853618.14; __utmz=213292672.1483861888.12.5.utmcsr=globalbioticinteractions.org|utmccn=(referral)|utmcmd=referral|utmcct=/' -H 'Proxy-Connection: keep-alive' --compressed > hierarchy_entries.tgz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
 78  301M   78  236M    0     0   400k      0  0:12:51  0:10:03  0:02:48  405k
curl: (18) transfer closed with 68777450 bytes remaining to read
```

**Comment by jhpoelen** [comment]
I was able to finally download the file using a separate server with more bandwidth (thanks to @godfoder and @mjcollin). Additionally, I created a data publication with the file at zenodo: [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.260489.svg)](https://doi.org/10.5281/zenodo.260489)
so that we can all enjoy uninterrupted citable downloads.
